app/src/main/java/com/github/se/signify/ui/Utils.kt

```diff
@@ -252,7 +252,7 @@ fun TopBar() {
       modifier =
           Modifier.fillMaxWidth()
               .height(5.dp)
-              .background(MaterialTheme.colorScheme.background)
+              .background(MaterialTheme.colorScheme.primary)
```

Comment:
```suggestion
              .background(MaterialTheme.colorScheme.background)
```
I believe this should stay as it is, as it's the reason we see the line at the top of the screens.

![image](https://github.com/user-attachments/assets/cda68a4f-275f-4f03-84ab-cad3e780358e)

Reply:
The `TopBar()` is just the color bar, so it has to be with the primary color to be seen as it is on the figma design from the beginning.

Reply:
Perhaps in this case, we should consider adding a "bottom line," as the screen doesn't look very appealing like this.